devtools/plugins/desktop/common/src/communication-layer/index.ts

```diff
@@ -27,7 +27,7 @@ export const useCommunicationLayer = (): Pick<
   });
 
   useEffect(() => {
-    // TODO:: figure out the connection errors when uncommenting this block
+    // TODO:: figure out the connection errors when uncommenting this block (#95)
```

Comment:
In my opinion this should be a prereq for merging this PR since it would _technically_ break existing capabilities if we didn't have support for flipper.

Reply:
Are we using Flipper atm? I had the impression the current implementation of dev tools wasn't in use.

Reply:
I would defer to @sugarmanz for a definitive answer but, at least as a capability, it is something that is possible.

Reply:
Flipper feature reestablished. The build for the Desktop client works leveraging `flipper-pkg` and the connection can be activated using the devtools popup.